### DISCUSSION TITLE
Show Likes/Dislikes made only by extension users

### DISCUSSION THREAD

**Discussion by ackvf** [Anarios/return-youtube-dislike#943] [Ideas]
This could better allow me to assess the like/dislike ratio of a video if I get only the verified responses from the same group of users that use the extension without youtube's own likes.

Either display those numbers on hover, or put a small number next to the already present number, or provide a second line below the buttons which, upon hover, would reveal the extension users' ratios.
![image](https://github.com/Anarios/return-youtube-dislike/assets/2313018/7f590a15-65b0-43c1-9779-348c01232fff)

**Comment by maokasaki** [comment]
I would love to contribute to the request. However, as of about an hour ago, the extension is no longer working. Firefox ver 118.0.2 (64 bit), Windows 10.

**Reply by maokasaki** [reply]
![image](https://github.com/Anarios/return-youtube-dislike/assets/83929895/6db2b22a-b750-4cf7-a527-bbd85071bcfa)

**Reply by ackvf** [reply]
This happens for all new videos until they collect enough data. Look up some old video, prior to 2022 or 2021 I think.

**Reply by maokasaki** [reply]
Their most recent update must've been pretty massive. I got an Adblocker warning from another extension I use as well. It broke everything, lol.

![image](https://github.com/Anarios/return-youtube-dislike/assets/83929895/933b2203-f776-4375-8e1d-74cb7b7e1fe3)

**Comment by three7s** [comment]
Same goes for Brave... Extension no longer working.

**Reply by three7s** [reply]
So in the new .11 version we are not getting the dislike ratio numbers back... OK? weird though.

**Comment by jpa102** [comment]
> Either display those numbers on hover

would it be like this? (i got the raw likes & dislikes from the api)

![image](https://github.com/user-attachments/assets/9df9f7ae-24ba-4fa6-b343-33c4964aadb6)

**Comment by Anarios** [comment]
Available since **4.0.3** in the side panel: it shows the raw likes and dislikes from RYD users separately. The latest release is [v4.0.5](https://github.com/Anarios/return-youtube-dislike/releases/tag/v4.0.5).

<img width="1897" height="867" alt="Raw like and dislike counts in the side panel" src="https://github.com/user-attachments/assets/a4206563-83d5-4e2a-aebd-1ebffa2f76af" />
